src/SignalR/server/Core/src/HubOptions.cs

```diff
@@ -51,5 +51,7 @@ public class HubOptions
         /// Gets or sets the max buffer size for client upload streams. The default size is 10.
         /// </summary>
         public int? StreamBufferCapacity { get; set; } = null;
+
+        internal List<IHubFilter> HubFilters { get; set; } = null;
```

Comment:
Is there some reason this isn't initialized by default? 

```suggestion
        internal List<IHubFilter> HubFilters { get;  } = new List<HubFilter>();
```

There's a bounded number of `HubOptions` instances so it's not saving much by doing this.

Reply:
No reason to allocate if it's never used